api/base/exceptions.py

```diff
@@ -1,5 +1,35 @@
-from rest_framework.exceptions import APIException
+
 from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+def json_api_exception_handler(exc, context):
+    """ Custom exception handler that returns errors object as an array """
+
+    from rest_framework.views import exception_handler
```

Comment:
It's a consequence of our integration of Django into the OSF. The error Dawn mentions happens when trying to load the OSF without also loading Django, as is the case with normal OSF operations.